test_libs/pyspec/eth2spec/test/phase_0/block_processing/test_process_attestation.py

```diff
@@ -398,3 +398,61 @@ def test_empty_aggregation_bits(spec, state):
     sign_attestation(spec, state, attestation)
 
     yield from run_attestation_processing(spec, state, attestation)
+
+
+@with_all_phases
+@spec_state_test
+def test_too_many_aggregation_bits(spec, state):
+    attestation = get_valid_attestation(spec, state, signed=True)
+    state.slot += spec.MIN_ATTESTATION_INCLUSION_DELAY
+
+    # one too many bits
+    attestation.aggregation_bits.append(0b0)
+
```

Comment:
Good to see tests. I was wondering about the signature correctness at my first look, because of the change in data after signing, but the aggregation bits don't influence it, so it is all fine.